subsys/net/ip/net_context.c

```diff
@@ -689,6 +689,53 @@ int net_context_put(struct net_context *context)
 	return 0;
 }
 
+/* If local address is not bound, bind it to INADDR_ANY and random port. */
+static int bind_default(struct net_context *context)
+{
+	sa_family_t af = net_context_get_family(context);
```

Comment:
Would prefer "family" instead of "af", as the family variable is used in other part of the code.